runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -4,33 +4,57 @@
 
 package akka.grpc
 
-import scala.concurrent.duration.Duration
+import akka.discovery.SimpleServiceDiscovery
+import akka.discovery.SimpleServiceDiscovery.{ Resolved, ResolvedTarget }
+import akka.grpc.internal.HardcodedServiceDiscovery
 
+import scala.concurrent.duration.{ Duration, FiniteDuration }
+import scala.collection.immutable
 import io.grpc.CallCredentials
+import scala.concurrent.duration._
 
 object GrpcClientSettings {
+  val DefaultResolveTimeout = 5.seconds
+  val DefaultPort = 443
```

Comment:
move to reference.conf, now that conf settings has been merged https://github.com/akka/akka-grpc/pull/254